### DISCUSSION TITLE
软件的界面无法显示了

### DISCUSSION THREAD

**Discussion by Sunsh4j** [localsend/localsend#1541] [L-简体中文]
刚在Windows上，装上最新版
结果退出一次后，现在界面 打不开了，无论是通过托盘图标，还是快捷方式，都无法打开

**Comment by hand50** [comment]
我也是

**Comment by Featastic** [comment]
如果是打开软件后有托盘程序，任务栏出现图标，但是预览窗口全白，也无法调出软件页面，可以尝试使用BCUninstaller卸载LocalSend，不要选静默，正常模式就行，卸载完后根据引导清理注册表项和settings.json文件（位于一个文件夹中,在BCUninstaller中该文件夹的置信度为“有问题”）。之后重新安装Localsend即可，但需要重新设置一遍。

**Comment by mrbai135234** [comment]
不会搞，我也是同样的问题，等更新吧

**Comment by Featastic** [comment]
确实，昨天用那个方法能行，然而今天开机后又bug了😅

FeatherOS v0.12
________________________________

**Comment by HSSkyBoy** [comment]
我的解决办法是把应用根目录下的Settings.json删掉就行，因为我试过修改，没用

**Reply by Featastic** [reply]
默认的位置在C:\用户\AppData\Roaming\LocalSend
不过更新后好像问题解决了

**Reply by Genie-Liu** [reply]
删除之后确实好使了

**Reply by Qee2** [reply]
更新之后，问题依旧。删除了json文件后，问题还在……

**Reply by kucoll** [reply]
真的有用，我9.16在官网下一个最新版，Settings.json里面是空内容打开也是没有界面，按你这个方法已删除果然可以了。。 这居然都能被你发现，牛批！

**Comment by HUMANLYMENG** [comment]
我这几天也突然这样， 我换了15.0版本还是一样的

**Comment by 1091515459** [comment]
我的也是这，删了setting还是不行

**Comment by 1091515459** [comment]
我问你们一个问题，你们的显卡都是Amd显卡么？

**Comment by 1091515459** [comment]
我在AMD显卡上有几率显示不出来但是英伟达显卡都正常

**Reply by hand50** [reply]
我的是amd,不过不知道是不是AMD的问题

**Comment by Featastic** [comment]
不是，Intel锐炬Xe/Nvidia GeForce 3050 Laptop
我不太觉得是显卡或者其驱动的问题

________________________________
发件人: zhangxu ***@***.***>
发送时间: 2025年1月7日星期二 11:22
收件人: localsend/localsend ***@***.***>
抄送: Featastic ***@***.***>; Comment ***@***.***>
主题: Re: [localsend/localsend] 软件的界面无法显示了 (Discussion #1541)


我问你们一个问题，你们的显卡都是Amd显卡么？

―

**Comment by cnfniw** [comment]
使用的是Windows7系统，重装后提示定位入口。看了一下函数ws2_32.dll，这个动态库根本就没有这个函数，现在测试的结题是升级1.16+后都是这个报错，回退到1.5+后就可以正常。看来必须是升级系统后才能支持了。放弃win7了。

**Reply by xCingular** [reply]
1.15.4可以用。
我是Windows11，之前还可以用的，昨天电脑打开发现app打不开，整了2天看到你的回复把1.17卸载了下了个1.15.4就正常使用了

**Comment by ARCJ137442** [comment]
已解决，使用Windows11，重新安装后仍然白屏无法调到前台。看了配置是这样的：

```json
{
    "flutter.ls_version": 2,
    "flutter.ls_show_token": "...",
    "flutter.ls_alias": "...",
    "flutter.ls_security_context": "...",
    "flutter.ls_color": "localsend",
    "flutter.ls_receive_history": [...],
    "flutter.ls_window_offset_x": 3271.0,
    "flutter.ls_window_offset_y": 1004.5,
    "flutter.ls_window_height": 500.0,
    "flutter.ls_window_width": 660.5,
    "flutter.ls_quick_save": false,
    "flutter.ls_quick_save_from_favorites": true
}
```

注意到
```json
    "flutter.ls_window_offset_x": 3271.0,
    "flutter.ls_window_offset_y": 1004.5,
```

这个「窗口偏移x」超出范围，联想了下可能是之前在其他屏幕运行导致的

关掉程序后把配置里边`window`相关的四条字段都删掉，重启后就正常了
